src/ToricVarieties.jl

```diff
@@ -85,61 +124,172 @@ export projective_space
 # 4: Properties
 ######################
 
-function is_normal_variety( v::NormalToricVariety )
+
+"""
+    is_normal_variety( v::normalToricVariety )
+
+Checks if the normal toric variety `v` is normal. (This function is somewhat tautological at this point.)
+
+# Examples
+```julia-repl
+julia> is_normal_variety( projective_space(  2 ) )
+true
+```
+"""
+function is_normal_variety( v::normalToricVariety )
     return true
 end
 export is_normal_variety
 
 
-function is_affine( v::NormalToricVariety )
+"""
+    is_affine( v::normalToricVariety )
+
+Checks if the normal toric variety `v` is affine.
+
+# Examples
+```julia-repl
+julia> is_normal_variety( projective_space(  2 ) )
+false
+```
+"""
+function is_affine( v::normalToricVariety )
     return v.polymakeNTV.AFFINE
 end
 export is_affine
 
 
-function is_projective( v::NormalToricVariety )
+"""
+    is_projective( v::normalToricVariety )
+
+Checks if the normal toric variety `v` is projective, i.e. if the fan of `v` is the the normal fan of a polytope.
+
+# Examples
+```julia-repl
```

Comment:
You could also use `jldoctest`, then these examples would also be run as part of the `doctest` step of the CI (provided that is set up), which has the advantage that the examples are tested, so one knows they actually are working and correct. But of course sometimes this is not desirable, e.g. if outputs are random or differ between Julia versions. Seems to be safe here, though?

Anyway, not important right now, but perhaps you'll want to change this down the road, so I thought I should mention it.

Reply:
Thank you for the suggestion. Sounds like a valuable addition!

For the time being (and in repeating the above strategy), I placed this in a separated issue: https://github.com/oscar-system/JToric.jl/issues/46.